src/components/ScoreBoard/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { convertSecondsToMMSS, getHighestScoreObj } from './../../utils/helpers';
+
+import './style.scss';
+
+const ScoreBoard = ({ scores }) => {
+  const highestScoreObj = getHighestScoreObj(scores);
+  return (
+    <div className="ScoreBoard">
+      <div className="title color-red">SCORE BOARD</div>
+      {!!scores && scores.length &&
+        <div className="score-container">
+          {scores.map(score => (
+            <div className="score color-white">
+              {highestScoreObj.id === score.id && <div className="color-red">PERSONAL BEST</div>}
+              {score.name} : {convertSecondsToMMSS(score.score)}
+            </div>
```

Comment:
Leverage destructuring 

```js
 {scores.map(({id, name, score}) => (
            <div className="score color-white">
              {highestScoreObj.id === id && <div className="color-red">PERSONAL BEST</div>}
              {name} : {convertSecondsToMMSS(score)}
            </div>
```

Reply:
Yes, thanks you for this :) 
how can I forget this